### ISSUE TITLE
Adjust Baro Altitude to QNH

### ISSUE THREAD

**Issue by DocTopi** [knicholson32/stratux_ahrs#16]
Great work until now !!! Is there any possibility for adjusting Baro altitude to QNH ?

**Comment by knicholson32** [comment]
That could certainly be done! Entering the altimeter setting into a pop-up would probably be the easiest solution. I'll have to think about how the user would summon the pop-up though- perhaps a long-press on the "Baro Altitude" button?

**Comment by knicholson32** [comment]
I ended up moving the AHRS alignment and gyro calibration to a sub-menu, so it was logical to put the altimeter setting in there as well. The feature currently exists on the develop branch; I'll do some testing to see if the conversion equations seem right when I get a chance.

**Comment by DocTopi** [comment]
The sub-menu for adjusting baro altitude to QNH is working very well. But in european aviation mbar is used instead of inHg. Is it possible to install in configuration.js a selection between inHg and mbar for entering baro values in mbar?

**Comment by knicholson32** [comment]
Should be very possible, just have to include a conversation for it. I'll reopen this and go from there. Thanks for the suggestion!

**Comment by knicholson32** [comment]
All finished adding the ability to use different units for the altimeter setting. In `configuration.js > altTape` you can change the unit to be `INHG`, `MB`, or `HPA`:
```javascript
/** ******************************* ALTITUDE ******************************* **/
var altTape = {
  units: UNITS.FEET,
  altimeter_setting_unit: UNITS.INHG, // <-- Unit can be INHG, MB, or HPA
  default_kollsman_inhg: 29.92,
  default_kollsman_millibar_hpa: 1013.2, 
  kollsman_memory_seconds: 18000
};
/** ******************************* ******** ******************************* **/
```

Let me know if you see any issues or have any suggestions. Thanks!

**Comment by DocTopi** [comment]
Hi Keenan,

 

please read my comment in the closed issue #20, and if you are in my opinion please open new issues, e.g.:

 

-          Portrait mode: splitting screen

-          Portrait mode upper section: Readability improvement

-          Portrait mode lower section: compass arc with heading bug and lateral course deviation

 

Thank you very much.

 

Kind regards 

Franz 

 

Dr. Franz Topitsch

Zahnarzt

Senselsweg 4

97645 Ostheim

Germany

E-mail: franz.topitsch@gmx.net

 

Von: Keenan Nicholson [mailto:notifications@github.com] 
Gesendet: Donnerstag, 2. Januar 2020 06:42
An: knicholson32/stratux_ahrs
Cc: DocTopi; Author
Betreff: Re: [knicholson32/stratux_ahrs] Adjust Baro Altitude to QNH (#16)

 

All finished adding the ability to use different units for the altimeter setting. In configuration.js > altTape you can change the unit to be INHG, MB, or HPA:

/** ******************************* ALTITUDE ******************************* **/
var altTape = {
  units: UNITS.FEET,
  altimeter_setting_unit: UNITS.INHG, // <-- Unit can be INHG, MB, or HPA
  default_kollsman_inhg: 29.92,
  default_kollsman_millibar_hpa: 1013.2, 
  kollsman_memory_seconds: 18000
};
/** ******************************* ******** ******************************* **/

Let me know if you see any issues or have any suggestions. Thanks!
